Added comment on parameterlength and extended the corresponding function in ANN.

benedict-96 opened the pull request:
Added `parameterlength` routine in `AbstractNeuralNetworks` and included this comment in the `sympnet_tutorial` file.

michakraus commented:
@benedict-96 The docs are failing to build because of a missing `parameterlength` method. Please check and fix.

benedict-96 commented:
Oh, of coure. Sorry. I added a method in `AbstractNeuralNetworks` (branch `main`) but that of course doesn't trigger a new release. Changed something in the script, the tests should go through now.
